src/main/java/io/fabric8/maven/docker/util/DockerEnvironment.java

```diff
@@ -0,0 +1,281 @@
+package io.fabric8.maven.docker.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.maven.plugin.MojoExecutionException;
+
+import io.fabric8.maven.docker.config.MachineConfiguration;
+
+/**
+ * launch docker-machine to obtain environment settings
+ */
+public class DockerEnvironment {
```

Comment:
Will add in two weeks when I return from vacation. 

Chas

> On Jun 26, 2016, at 6:32 PM, Roland Huß <notifications@github.com> wrote:
> 
> In src/main/java/io/fabric8/maven/docker/util/DockerEnvironment.java:
> 
> > +import java.util.concurrent.Callable;
> > +import java.util.concurrent.ExecutionException;
> > +import java.util.concurrent.ExecutorService;
> > +import java.util.concurrent.Executors;
> > +import java.util.concurrent.Future;
> > +import java.util.concurrent.TimeUnit;
> > +import java.util.concurrent.TimeoutException;
> > +
> > +import org.apache.maven.plugin.MojoExecutionException;
> > +
> > +import io.fabric8.maven.docker.config.MachineConfiguration;
> > +
> > +/**
> > + * launch docker-machine to obtain environment settings
> > + */
> > +public class DockerEnvironment {
> Some unit tests for this class would be nice, since its quite complex.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>